core/src/main/kotlin/fr/sncf/osrd/api/api_v2/standalone_sim/SimulationResponse.kt

```diff
@@ -22,9 +22,20 @@ class SimulationSuccess(
     val provisional: ReportTrain,
     @Json(name = "final_output") val finalOutput: CompleteReportTrain,
     val mrsp: MRSPResponse,
-    @Json(name = "power_restrictions") val powerRestrictions: List<PowerRestriction>,
+    @Json(name = "electrical_profiles") val electricalProfiles: ElectricalProfiles,
```

Comment:
This is `electricalProfiles: RangeValues<ElectricalProfileValue>`. However, `RangeValues` is a list of distances, but they are indeed path offsets everywhere. Could you modify its definition in `CommonSchemas.kt` and its uses to have a list of offsets everywhere ? Also noticed the class's definition is duplicated in `PathPropResponse.kt`, it should be removed and instead use its definition in `CommonSchemas.kt`.